bin/create-or-update-database-roles.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# -----------------------------------------------------------------------------
+# Script that invokes the database role-manager AWS Lambda function to create
+# or update the Postgres user roles for a particular environment.
+# The Lambda function is created by the infra/app/database root module and is
+# defined in the infra/app/database child module.
+#
+# Positional parameters:
+#   APP_NAME (required) – the name of subdirectory of /infra that holds the
+#     application's infrastructure code.
+#   ENVIRONMENT (required) - the name of the application environment (e.g. dev
+#     staging, prod)
+# -----------------------------------------------------------------------------
+set -euo pipefail
+
+APP_NAME=$1
+ENVIRONMENT=$2
+
+./bin/terraform-init.sh infra/$APP_NAME/database $ENVIRONMENT
+DB_ROLE_MANAGER_FUNCTION_NAME=$(terraform -chdir=infra/$APP_NAME/database output -raw role_manager_function_name)
+
+echo "================================"
+echo "Creating/updating database users"
+echo "================================"
+echo "Input parameters"
+echo "  APP_NAME=$APP_NAME"
+echo "  ENVIRONMENT=$ENVIRONMENT"
+echo
+echo "Invoking Lambda function: $DB_ROLE_MANAGER_FUNCTION_NAME"
+aws lambda invoke --function-name $DB_ROLE_MANAGER_FUNCTION_NAME response.json > /dev/null
```

Comment:
but yeah i can remove the `> /dev/null`, i realize that the issue i was having could be solved with `--no-cli-pager`. without that option, the aws command would wait for user input before proceeding with the script and since i didn't know about that option i used `> /dev/null` as a hack